packages/shared/src/enums/Tier.ts

```diff
@@ -0,0 +1,5 @@
+export enum Tier {
+  Free = '0',
+  Launch = '1',
+  Growth = '2',
+}
```

Comment:
This will infer `number` types not so?

```suggestion
export enum Tier {
  Free,
  Launch,
  Growth,
}
```

Reply:
Yessir and they should be numbers not string 😅  good catch I was trying stuff out and forgot to swap back